content/en/users/data/management/datahub/links/_index.md

```diff
@@ -1,9 +1,10 @@
 ---
-title: "Links"
-linkTitle: "Links"
-type: docs
-description: "Related links"
+title: DataHub Links
```

Comment:
I feel this page is not bringing much, although I foresee similar ones coming for other services as we improve the documentation. For sure I would not call it "Links", maybe something like "Extra information"?

Reply:
Agree. I left it there, for future improvement. This PR is already doing too much.